resources/views/store/index_registration.blade.php

```diff
@@ -4,50 +4,18 @@
 
 @section('content')
 
-@include('store.partials.navbar', ['headerTitle' => 'Search for a family'])
+@include('store.partials.navbar', ['headerTitle' => 'Search for a ' . ($programme ? 'household' : 'family')])
```

Comment:
Could be Family::getAlias($programme) for future proofing

Reply:
> Do we use fuzzy search elsewhere? can we remove it?

@charlesstrange2 I'm going to write a new ticket for that, rather than mess around with composer in this one.